src/main/java/ru/sbt/mipt/oop/objects/Door.java

```diff
@@ -0,0 +1,29 @@
+package ru.sbt.mipt.oop.objects;
+
+import ru.sbt.mipt.oop.eventhandlers.SmartDoorEventHandler;
+import ru.sbt.mipt.oop.events.SensorEvent;
+
+public class Door {
+    private final String id;
+    private boolean isOpen;
+    private String placeName;
+
+    public Door(boolean isOpen, String id, String placeName) {
+        this.isOpen = isOpen;
+        this.id = id;
+        this.placeName = placeName;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getPlaceName() { return placeName; }
+
+    public void handleUsualEvent(SensorEvent event) {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%B4%D0%B5%D0%BB%D0%B5%D0%B3%D0%B8%D1%80%D1%83%D0%B9%D1%82%D0%B5-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D1%83-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D1%82%D0%B4%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%BC-%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B0%D0%BC